components/chat.tsx

```diff
@@ -118,7 +119,6 @@ const Chat: React.FC<ScriptProps> = ({
     setMessages((prevMessages) => [
       ...prevMessages,
       { type: MessageType.User, message },
-      { type: MessageType.Agent, message: 'Waiting for model response...' },
```

Comment:
Removing this does mean that a user won't get any immediate feedback upon hitting enter. Prior to this PR we were pushing this message to simulate more immediate feedback. Any way that we can bring that back in some variant?

Reply:
It was added somewhere else.

Reply:
I was bringing this up because I see this behavior has regressed in my test, there's a big delay when talking to Tildy before I receive a "Waiting for model response..." like from a few days ago.

Reply:
Not seeing that. Let me retest.

Reply:
It has to be the first chat with something large like Tildy.

Reply:
@tylerslaton I made changes on for this. Let me know what you think.

Reply:
Yep much better!